app/controllers/api/v1/merchants/random_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Merchants::RandomController < ApplicationController
+
+  def show
+    render json: Merchant.order("RANDOM()").first
```

Comment:
this logic could be moved to a method on your `Merchant` model. the controller doesn't need to do the logic to find the correct merchant 👍